rundeckapp/grails-spa/packages/ui-trellis/src/app/components/job/resources/tests/NodeDetailSimple.spec.ts

```diff
@@ -81,34 +81,27 @@ describe("NodeDetailsSimple Component", () => {
         nodeColumns,
         showExcludeFilterLinks: true,
       });
-      // General attributes
-      const generalAttributes = {
-        description: "Test Node 1",
-        "ui:status:text": "Healthy",
-        "ui:status:icon": "fa-check",
-        osFamily: "Linux",
-        osName: "Ubuntu",
-        osVersion: "20.04",
-        osArch: "x86_64",
-      };
-      Object.entries(generalAttributes).forEach(([key, value]) => {
-        const attributeElement = wrapper.find(`td.value:contains(${value})`);
-        expect(attributeElement.exists()).toBe(true);
-      });
-      // Attributes without namespaces
-      const attributesWithNoNamespaces = {
-        key3: "value3",
-        key4: "value4",
-        key5: "value5",
-        key6: "value6",
-      };
-      Object.entries(attributesWithNoNamespaces).forEach(([key, value]) => {
-        const attributeElement = wrapper.find(`td.setting:contains(${value})`);
-        expect(attributeElement.exists()).toBe(true);
-      });
+
+      expect(wrapper.find(`[data-test="description"]`).text()).toBe(
+        "Test Node 1",
+      );
+      expect(wrapper.find(`[data-testid="node-status-text"]`).text()).toBe(
+        "Healthy",
+      );
+      const icon = wrapper.find(`i.fa-check.node-status-icon`);
+      expect(icon.exists()).toBeTruthy();
+
+      const osAttributes = ["Linux", "Ubuntu", "20.04", "x86_64"];
+      const osElements = wrapper.findAll("td[data-test='os-data'] > a");
+      expect(osElements.length).toBe(4);
+      osElements.forEach((osElement) =>
+        expect(osAttributes).toContain(osElement.text()),
+      );
+
```

Comment:
it seems the attribute value tests 'key3' through 'key6' have not been converted, so the new test does not check for the presence of those values

Reply:
oops, i didn't read enough, looks good